services/comfy/entrypoint.sh

```diff
@@ -44,4 +44,16 @@ for to_path in "${!MOUNTS[@]}"; do
   echo Mounted $(basename "${from_path}")
 done
 
+if [ "$(ls -A /stable-diffusion/custom_nodes)" ]; then
+  chmod 777 -R "/stable-diffusion/custom_nodes/"
```

Comment:
why is this necessary?

Reply:
Testing reasons. I wasn't going to bother fixing everything until there was a proposed solution